src/main/java/org/broadinstitute/consent/http/service/users/handler/DACUserRolesHandler.java

```diff
@@ -304,10 +305,6 @@ private void changeRolesWithoutDelegation(DACUser updatedUser, List<UserRole> re
         return roles.stream().filter(rol -> !toSubstractRolesNames.contains(rol.getName().toUpperCase())).collect(Collectors.toList());
     }
 
-    private void removeRole(Integer dacUserId, String role) {
-        userRoleDAO.removeSingleUserRole(dacUserId, roleIdMap.get(role));
```

Comment:
This was the root of the bug we saw where users were not being removed. Accessing the string name via the `roleIdMap` was brittle so refactored to be more clear about (and consistent using the `UserRoles` enum) where the ID is coming from.